Makefile_windows

```diff
@@ -1,67 +1,88 @@
-.PHONY: dev lint complex coverage pre-commit sort deploy destroy deps unit infra-tests integration e2e coverage-tests docs lint-docs build format
+.PHONY: deploy destroy format lint install build tests unit_tests dev_deploy docs
+.DEFAULT_GOAL := help
```

Comment:
no, i wanted you to take the current Makefile file, dont change it. use it to fix the Makefile_windows file to have the same features as the Makefile one